docs/source/how-to/migrations/upgrade-to-16.md

```diff
@@ -50,21 +33,34 @@ This request is sent to {attr}`.OAuthenticator.userdata_url` in {meth}`.OAuthent
    Previously, a GenericOAuthenticator only trait.
    ```
 
-4. The method used for sending the `access token` in the [userdata request](userdata_request), called {attr}`.OAuthenticator.userdata_token_method`, is now a configurable feature of _all the oauthenticators_.
+4. The method used for sending the `access token` in the userdata request[^userdata_request], called {attr}`.OAuthenticator.userdata_token_method`, is now a configurable feature of _all the oauthenticators_.
 
    ```{note}
    Previously, a GenericOAuthenticator only trait.
    ```
 
-5. It is now possible to pass extra parameters to the [token request](token_request), using {attr}`.OAuthenticator.token_params` for _all of the oauthenticators_.
+5. It is now possible to pass extra parameters to the token request[^token_request], using {attr}`.OAuthenticator.token_params` for _all of the oauthenticators_.
 
    ```{note}
    Previously, a GenericOAuthenticator only trait.
    ```
 
-6. It is now possible to set whether or not to use basic authentication for the access [token request](token_request) using {attr}`.OAuthenticator.basic_auth` for _all of the oauthenticators_.
+6. It is now possible to set whether or not to use basic authentication for the access token request[^token_request] using {attr}`.OAuthenticator.basic_auth` for _all of the oauthenticators_.
    Currently it defaults to `False`.
 
    ```{note}
    Previously, a GenericOAuthenticator only trait.
    ```
+
+[^token_request]: **The token request.**
+
+    Whenever _token request_ is used, it refers to the HTTP request exchanging the [oauth code](https://www.rfc-editor.org/rfc/rfc6749#section-1.3.1) for the [access token](https://www.rfc-editor.org/rfc/rfc6749#section-1.4).
```

Comment:
I consider a section heading saying "foot notes" or similar, feel free to ignore - uncertain what I think.